packages/x-node-jsx/readme.md

```diff
@@ -0,0 +1,58 @@
+# x-node-jsx
+
+This module extends Node's `require()` function to enable the use of `.jsx` files at runtime. It uses [Pirates] to safely add a require hook and [Sucrase] to transform code on-the-fly.
+
+[Pirates]: https://github.com/ariporad/pirates
+[Sucrase]: https://github.com/alangpierce/sucrase
+
+
+## Installation
+
+This module is compatible with Node 8+ and is distributed on npm.
+
+```bash
+npm install -S @financial-times/x-node-jsx
+```
+
+To install the `require` hook you must include and call the `x-node-jsx` module:
+
+```js
+require('@financial-times/x-node-jsx')();
+```
+
+The `x-node-jsx` install function will return a new function which can be used to later remove the `.jsx` file hook:
+
+```js
+const installHook = require('@financial-times/x-node-jsx');
+
+const removeHook = installHook();
+
+removeHook();
+```
+
+## Usage
+
+This package provides a single helper function to. An options object may be provided to the function when called. The options and their defaults are shown below:
```

Comment:
"...provides a single helper function to" - to what? :neutral_face:

Reply:
"This package _even_ provides a single helper function, too!"